content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -7,40 +11,24 @@ options(
   xpinch = 300,
   ypinch = 300,
   yaml.eval.expr = TRUE,
-  usethis.full_name = "Research Institute for Nature and Forest",
-  usethis.description = list(
-    `Authors@R` = "c(
-  person(
-    \"Voornaam\", \"Achternaam\", role = c(\"aut\", \"cre\"),
-    email = \"voornaam.achternaam@inbo.be\",
-    comment = c(ORCID = \"9999-9999-9999-9999\")),
-  person(
-    \"Research Institute for Nature and Forest\",
-    role = c(\"cph\", \"fnd\"), email = \"info@inbo.be\"))",
-    License = "GPL-3",
-    Language = "en-GB",
-    Encoding = "UTF-8",
-    Roxygen = "list(markdown = TRUE)"
-  ),
+  lintr.linter_file = system.file("lintr", package = "checklist"),
```

Comment:
This forces everyone to follow the linter rules as specified in the checklist package if I understand correctly? (Does this overwrite the linters that are shown interactively and by default in RStudio?) 
I think this is a good idea.

Reply:
People who want to use them can still add them manually to the `Rprofile.site`. They would have to edit the file anyway to replace the placeholder values for the author names with their real name.

Reply:
Yes it will affect the interactive checks in RStudio and manual runs of lintr outside of checklist.